example/.env.example

```
@@ -1,2 +1,3 @@
 GR4VY_ID=spider
-TOKEN=
\ No newline at end of file
+PRIVATE_KEY=
```

We have the pattern of prefixing env vars with GR4VY_ in other projects

```suggestion
GR4VY_PRIVATE_KEY=
```